src/Persistence/Migrations/20240103030850_AddRestrictionPublicReason.Designer.cs

```diff
@@ -0,0 +1,2182 @@
+﻿// <auto-generated />
+using System;
+using Crpg.Domain.Entities;
+using Crpg.Domain.Entities.ActivityLogs;
+using Crpg.Domain.Entities.Battles;
+using Crpg.Domain.Entities.Characters;
+using Crpg.Domain.Entities.Clans;
+using Crpg.Domain.Entities.Items;
+using Crpg.Domain.Entities.Parties;
+using Crpg.Domain.Entities.Restrictions;
+using Crpg.Domain.Entities.Settlements;
+using Crpg.Domain.Entities.Users;
+using Crpg.Persistence;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using NetTopologySuite.Geometries;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace Crpg.Persistence.Migrations
+{
```

Comment:
Scoped namespace namespace Crpg.Persistence.Migrations;

Reply:
Still not scoped?

Reply:
fix fix fix )